stacked-predictions/readme.md

```diff
@@ -0,0 +1,20 @@
+# Stacked Predictions
+
+This script takes two datasets, a training and a holdout, and builds
+several predictors using the training dataset. These include a model,
+a bagging ensemble, a random decision forest, a boosted tree, and a
+logistic regression (for categorical datasets).  The script then runs
```

Comment:
it'd be really nice to take a parameter that specified the kinds of models we want to use (a list of strings: `["model", "ensemble", "ensemble", "ensemble", "logisticregression"]` by default), and a list of maps with creation parameters (for instance, `[{}, {"random": true, "number_of_models":5}, ..]`)